README.md

```diff
@@ -17,155 +17,94 @@ This is the source code for the VESC DC/BLDC/FOC controller. Read more at
-2. Locate and Download version **gcc-arm-none-eabi-7-2018-q2** for your machine  
-   ```GNU Arm Embedded Toolchain: 7-2018-q2-update     June 27, 2018```  
-   Linux 64-bit version can be downloaded from [here](https://developer.arm.com/-/media/Files/downloads/gnu-rm/7-2018q2/gcc-arm-none-eabi-7-2018-q2-update-linux.tar.bz2?revision=bc2c96c0-14b5-4bb4-9f18-bceb4050fee7?product=GNU%20Arm%20Embedded%20Toolchain,64-bit,,Linux,7-2018-q2-update)  
-3. Unpack the archive in the file manager by right-clicking on it and select "extract here"
-4. Change directory to the unpacked folder, unpack it in /usr/local by execute the following command
-   ```
-   cd gcc-arm-none-eabi-7-2018-q2-update-linux  
-   sudo cp -RT gcc-arm-none-eabi-7-2018-q2-update/ /usr/local  
-   ```
+### On Ubuntu (Linux)/macOS
+- Tools: `git`, `wget, `make`, `python3`, and `pip`
+- Additional Linux requirements: `libgl-dev` and `libxcb-xinerama0`
+- Helpful macOS tools: 
 
-**Method 2 - Through apt install**
 ```bash
-sudo add-apt-repository ppa:team-gcc-arm-embedded/ppa
-sudo apt update
-sudo apt install gcc-arm-embedded
+brew install stlink
+brew install openocd
 ```
 
+### On Windows
+- MS Visual Compiler
+- Chocolately: https://chocolatey.org/install
+- Powershell: https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows
+- Git: https://git-scm.com/download/win. Make sure to click any boxes to add Git to your Environment (aka PATH)
+- Python 3: https://www.python.org/downloads/. Make sure to click the box to add Python3 to your Environment.
 
-**Optional - Add udev rules to use the stlink v2 programmer without being root**
-```bash
-wget vedder.se/Temp/49-stlinkv2.rules
-sudo mv 49-stlinkv2.rules /etc/udev/rules.d/
-sudo udevadm trigger
-```
+## Install Dev environment and build
 
-### On MacOS
+### On Ubuntu (Linux)/MacOS
+Open up a terminal
+1. `pip install aqtinstall`
+5. `git clone http://github.com/vedderb/bldc.git`
+6. `cd bldc`
+6. Continue with [On all platforms](#on-all-platforms)
 
-Go to the [GNU ARM embedded toolchain downloads Website](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads) and select the mac version, download it and extract it to your user directory.
+### On Windows
 
-Append the bin directory to your **$PATH**. For example:
+3. Open up a Powershell terminal (Resist the urger to run Powershell as administrator, that will break things)
+4. Type `choco install make`
+4. `pip install aqtinstall`
+5. `git clone http://github.com/vedderb/vesc_tool`
+6. `cd bldc`
+6. Unzip the file at https://discordapp.com/channels/904830990319485030/910181652192448532/917451210791989248 into this directory
+6. Continue with [On all platforms](#on-all-platforms)
 
+### On all platforms
 
-```bash
-export PATH="$PATH:/Users/your-name/gcc-arm-none-eabi-8-2019-q3-update/bin/"
-```
+7. `git checkout origin/master`
+8. `make arm_sdk_install`
+9. `make qt_install`
```

Comment:
qt_install is probably good to move to an optional section as it isn't needed unless you want to build vesc tool or use qt creator. pip install aqtinstall can be moved there too. Seeing the pip-command in build instructions always puts me in fight-against-python-dependency-hell-mode.

Reply:
It sets up Qt Creator for firmware compilation. That's pretty handy if people don't already have an IDE that they prefer.

Reply:
Since Qt is necessary for the VESC-Tool, it makes sense to me to have it be part of a standard setup.

Reply:
I think it is quite common to just compile the firmware and use pre-compiled vesc tool, so it makes more sense to have them separate. Especially since qt is so huge compared to the other tools needed. I also wonder if this script has a way to get qt 5.12 as that is what I'm using, and if it gets all qt modules correctly.

Reply:
@vedderb fixed.